packages/protocol/test/MoxieBondingCurve.ts

```diff
@@ -1,4745 +1,4753 @@
 import { loadFixture } from "@nomicfoundation/hardhat-network-helpers";
 import { expect } from "chai";
-import hre, { ethers } from "hardhat";
-import { SubjectERC20 as SubjectERC20Type } from "../typechain-types";
+import hre, { ethers, upgrades } from "hardhat";
+import type { SubjectERC20 as SubjectERC20Type } from "../typechain-types";
 import {
-    getExpectedSellReturnAndFee,
-    getExpectedBuyAmountAndFee,
+  getExpectedSellReturnAndFee,
```

Comment:
I think you can create a new test file `MoxieBondingCurveUpgradeTest.ts` which covers upgradability test

Reply:
yeah rebased done, I've moved all the upgradability tests to newly created files